Sources/SwiftSyntaxBuilder/gyb_generated/BuildableNodes.swift

```diff
@@ -13252,20 +12916,6 @@ public struct TuplePattern: PatternBuildable, ExpressibleAsTuplePattern {
     assert(rightParen.text == ")")
   }
 
-  /// A convenience initializer that allows:
-  ///  - Initializing syntax collections using result builders
-  ///  - Initializing tokens without default text using strings
-  public init(
-    leftParen: TokenSyntax = TokenSyntax.`leftParen`,
-    rightParen: TokenSyntax = TokenSyntax.`rightParen`,
-    @TuplePatternElementListBuilder elementsBuilder: () -> ExpressibleAsTuplePatternElementList = { TuplePatternElementList([]) }
```

Comment:
Please keep this as `TuplePatternElement` conforms to `HasTrailingComma`

Reply:
✅ Similar to `TupleTypeElement`.